sdk/eventhub/event-hubs/src/eventData.ts

```diff
@@ -197,22 +197,22 @@ export interface ReceivedEventData {
   /**
    * @property {Dictionary<any>} [applicationProperties] The application specific properties.
    */
-  properties?: { [key: string]: any };
+  properties: { [key: string]: any };
```

Comment:
properties should be optional.
When user sends `EventData` without properties, this will not be set on the returned event as well

Reply:
Ohh yesss!!